recipes-openxt/sync-client/sync-client_git.bb

```diff
@@ -33,7 +33,19 @@ S = "${WORKDIR}/git"
 inherit distutils3
 inherit update-rc.d
 
-FILES_${PN} += "/etc/init.d/${INITSCRIPT_NAME}"
+PACKAGES += "${PN}-synccmd"
+
+FILES_${PN} = " \
+	${bindir}/sync-client \
+	${bindir}/sync-client-daemon \
+	${PYTHON_SITEPACKAGES_DIR}/sync_client/* \
+	/etc/init.d/${INITSCRIPT_NAME}"
+
+FILES_${PN}-synccmd = " \
+	${bindir}/pysynccmd \
+	${bindir}/sync-cmd \
+	${PYTHON_SITEPACKAGES_DIR}/pysynchronizer/* \
+	/etc/init.d/${INITSCRIPT_NAME}"
```

Comment:
This INITSCRIPT_NAME isn't needed for synccmd.  

Also, please put the closing " at the start of a newline:
```
FOO = " \
    BAR \
"
```
Here and above.

Reply:
Ugh, a little copy/paste cruft, will clean up.